commit d247110e60d3ba009c038a74e1f2b70bb1ba6a01
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Wed Aug 20 00:24:19 2014 +0200

    Add hwloc-ls.desktop file for start menu integration in gnome/KDE

diff --git a/utils/Makefile.am b/utils/Makefile.am
index 872d342..e8743a7 100644
--- a/utils/Makefile.am
+++ b/utils/Makefile.am
@@ -1,5 +1,5 @@
 # Copyright © 2009-2013 Inria.  All rights reserved.
-# Copyright © 2009-2012 Université Bordeaux 1
+# Copyright © 2009-2012, 2014 Université Bordeaux 1
 # Copyright © 2009-2011 Cisco Systems, Inc.  All rights reserved.
 #
 # See COPYING in top-level directory.
@@ -122,6 +122,9 @@ if HWLOC_HAVE_LINUX
 nodist_man_MANS += $(hgt_page)
 endif HWLOC_HAVE_LINUX
 
+APPLICATIONSdir = @datarootdir@/applications
+dist_APPLICATIONS_DATA = hwloc-ls.desktop
+
 .1in.1:
 	@ echo Creating $@ man page...
 	@ $(SED) -e 's/#PACKAGE_NAME#/@PACKAGE_NAME@/g' \
diff --git a/utils/hwloc-ls.desktop b/utils/hwloc-ls.desktop
new file mode 100644
index 0000000..5cf0ad5
--- /dev/null
+++ b/utils/hwloc-ls.desktop
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Hardware Locality ls
+Name=Show hardware topology
+Exec=hwloc-ls
+Terminal=false
+Type=Application
+Categories=Application;System;
+Keywords=System;Utility;
